### PR TITLE
Public client displays images given by the API

### PR DESCRIPTION
**Pull request by christerforsgren91**
PT: https://www.pivotaltracker.com/story/show/178194650

### User Story
```
As a website
In order to make the articles more visually appealing
I need to display the images received from the API
```

### Changes Proposed 
-Add image to article

### Video 
https://recordit.co/4QT57v5mf2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "start": "PORT=3001 react-scripts start ",
-    "build": "react-scripts build",
+    "build": "react-scripts build && echo '/* /index.html 200' | cat >build/_redirects",
```
Comment (DavDan1) [inline package.json:22]: what does this do?

File: src/components/Article.jsx [hunk 2]
```diff
@@ -15,13 +15,19 @@ const Article = () => {
...
+            style={{ width: '100%' }}
+          />
           <h1 data-cy='article-title' className='article-title'>
             {article.title}
           </h1>
           <p data-cy='article-author' className='article-author'>
             {article.author &&
               `Written by: ${article.author.first_name} ${article.author.last_name}`}
-            :{' '}
+            {' '}-{' '}
```
Comment (offcast-dev) [inline src/components/Article.jsx:30]: what that for?

File: src/components/MainPage.jsx [hunk 3]
```diff
@@ -20,11 +19,11 @@ const MainPage = () => {
 
   return (
     <>
-    <CategoryMenu />
-    {error && (
-      <Header data-cy='error-message' color='red'>
-      {message}
-      </Header>
+      <CategoryMenu />
```
Comment (DavDan1) [inline src/components/MainPage.jsx:22]: standartilise prettier?

**Review by offcast-dev — APPROVED** [review]
just 1 small question

File: src/style.css [hunk 4]
```diff
@@ -166,7 +166,7 @@ body {
   background-color: #202325;
   align-content: center;
   margin: 100px;
-  padding: 100px 300px 100px 300px;
+  padding: 5%;
```
Comment (DavDan1) [inline src/style.css:169]: Good call

**Review by DavDan1 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (lukeperrin10) [reply package.json:22]: This helps netlify redirect to correct page. Instead of rendering an error.

File: src/components/MainPage.jsx (hunk 3 above)
Comment (lukeperrin10) [reply src/components/MainPage.jsx:22]: This will be fixed in next PR.

File: src/components/Article.jsx (hunk 2 above)
Comment (lukeperrin10) [reply src/components/Article.jsx:30]: This is adding spaces between name and date. But for next PR we can string interpolate for this instead.

Comment (offcast-dev) [reply src/components/Article.jsx:30]: yes that would be better

**Review by 1CIM — APPROVED** [review]
Looks good to me 👍

File: src/components/Article.jsx (hunk 2 above)
Comment (tochman) [reply src/components/Article.jsx:30]: Okay...

**Pull request merged**